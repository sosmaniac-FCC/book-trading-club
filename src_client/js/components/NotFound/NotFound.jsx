import React from 'react';

const NotFound = () => (
    <div className="row" id="not-found">
        <h1 className="col s12 center">Uh oh! 404 Page not found...</h1>
    </div>
);

export default NotFound;