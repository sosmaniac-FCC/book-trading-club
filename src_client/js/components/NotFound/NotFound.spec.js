// expect from jest_expect.js
/* global expect */

import React from 'react';
import { shallow, configure } from 'enzyme';
import enzymeAdapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';

import NotFound from './NotFound.jsx';

configure({ adapter: new enzymeAdapter() });

test('An invalid route loads a page indicating a status of 404 as expected.', () => {
    const component = shallow(<NotFound />);
    const tree = toJSON(component);
    expect(tree).toMatchSnapshot();
});

test('Status of 404 screen should have no style attributes.', () => {
    const component = shallow(<NotFound />);
    expect(component.find('[style]')).toHaveLength(0);
});