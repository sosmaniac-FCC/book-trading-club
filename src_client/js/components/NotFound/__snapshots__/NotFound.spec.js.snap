// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`An invalid route loads a page indicating a status of 404 as expected. 1`] = `
<div>
  <h1>
    Uh oh! 404 Page not found...
  </h1>
</div>
`;
